seatunnel-connectors-v2/connector-doris/src/test/java/org/apache/seatunnel/connectors/doris/catalog/PreviewActionTest.java

```diff
@@ -110,10 +110,20 @@ private void assertPreviewResult(
             Catalog.ActionType actionType,
             String expectedSql,
             Optional<CatalogTable> catalogTable) {
-        PreviewResult previewResult =
-                catalog.previewAction(
-                        actionType, TablePath.of("testddatabase.testtable"), catalogTable);
-        Assertions.assertInstanceOf(SQLPreviewResult.class, previewResult);
-        Assertions.assertEquals(expectedSql, ((SQLPreviewResult) previewResult).getSql());
+        if (!actionType.equals(Catalog.ActionType.CREATE_TABLE)) {
+            PreviewResult previewResult =
+                    catalog.previewAction(
+                            actionType, TablePath.of("testddatabase.testtable"), catalogTable);
+            Assertions.assertInstanceOf(SQLPreviewResult.class, previewResult);
+            Assertions.assertEquals(expectedSql, ((SQLPreviewResult) previewResult).getSql());
+        } else {
+            Assertions.assertThrows(
+                    RuntimeException.class,
+                    () ->
+                            catalog.previewAction(
+                                    actionType,
+                                    TablePath.of("testddatabase.testtable"),
+                                    catalogTable));
+        }
```

Comment:
The file main to test preview create table action. So I think we should modify test to make sure the test case of preview create table action passed, not throw exception. I think we can add primary keys in `CATALOG_TABLE`.